docs/documentation/e2e.md

```diff
@@ -16,14 +16,10 @@ End-to-end tests are run via [Protractor](https://angular.github.io/protractor/)
 ## Options
 `--config` (`-c`) use a specific config file. Defaults to the protractor config file in `.angular-cli.json`.
 
-`--specs` (`-sp`) override specs in the protractor config.
-Can send in multiple specs by repeating flag (`ng e2e --specs=spec1.ts --specs=spec2.ts`).
+`--proxy-config` (`-pc`) Proxy configuration file.
 
-`--element-explorer` (`-ee`) start Protractor's
-[Element Explorer](https://github.com/angular/protractor/blob/master/docs/debugging.md#testing-out-protractor-interactively)
-for debugging.
+`--serve` Compile and Serve the app. All non-reload related serve options are also available (e.g. --port=4400).
```

Comment:
The live-reload options are still available, it's just that it defaults to false.